src/server/debug/server/server.go

```diff
@@ -122,11 +123,44 @@ func (s *debugServer) handleRedirect(
 					return err
 				}
 			}
+
```

Comment:
nit, I tend to prefer not having arbitrary spaces.

Reply:
ah sure, I initially had the function inline and I guess I left this when I moved it